plugins/woocommerce-admin/client/products/product-page.tsx

```diff
@@ -14,6 +17,13 @@ const dummyProduct = {
 	short_description: 'Short product description content',
 } as Product;
 
+declare const productBlockEditorSettings: ProductEditorSettings;
```

Comment:
Could we simplify this naming since we don't use blocks in the other component names?

```suggestion
declare const productEditorSettings: ProductEditorSettings;
```